rts/Makefile

```diff
@@ -136,9 +140,20 @@ _build/wasm32:
 # Common configuration
 # 
 
-RTS_EMSCRIPTEN_64=emcc -c --target=wasm64-unknown-emscripten -fpic
-RTS_EMSCRIPTEN_64_DEBUG=$(RTS_EMSCRIPTEN_64)
-RTS_EMSCRIPTEN_64_RELEASE=$(RTS_EMSCRIPTEN_64) -O2
+# Arguments obtained by Emscripten 3.1.64:
+# `emcc -c --target=wasm64-unknown-emscripten -fpic --verbose`
+# while skipping unneeded arguments.
+RTS_LL_64=$(CLANG) -cc1 -triple wasm64-unknown-emscripten -emit-obj \
+	-mrelocation-model pic -pic-level 1 -mframe-pointer=none \
+	-ffp-contract=on -fno-rounding-math -mconstructor-aliases \
+	-target-feature +mutable-globals -fvisibility=hidden \
+	-Werror=implicit-function-declaration \
+	-iwithsysroot/include/fakesdl -iwithsysroot/include/compat \
+	-mllvm -combiner-global-alias-analysis=false \
+	-mllvm -enable-emscripten-sjlj -mllvm -disable-lsr \
+	-Wno-override-module
+RTS_LL_64_DEBUG=$(RTS_LL_64)
+RTS_LL_64_RELEASE=$(RTS_LL_64) -Oz
```

Comment:
Could it be that the change from 02 to 0z is causing the rts size increase?

Reply:
I believe the increase would be even higher with `O2`. I'll double check and let you know.

Reply:
I removed `emscripten` by dedicated Rust targets that generate PIC shared Wasm libraries.